hudi-common/src/test/java/org/apache/hudi/common/util/TestParquetUtils.java

```diff
@@ -132,7 +132,7 @@ private void writeParquetFile(String filePath, List<String> rowKeys) throws Exce
     BloomFilter filter = BloomFilterFactory
         .createBloomFilter(1000, 0.0001, 10000, bloomFilterTypeToTest);
     HoodieAvroWriteSupport writeSupport =
-        new HoodieAvroWriteSupport(new AvroSchemaConverter().convert(schema), schema, filter);
+        new HoodieAvroWriteSupport(new AvroSchemaConverter().convert(schema), schema, filter, false);
```

Comment:
don't we need to test both true and false?

Reply:
Added